components/sharing-dialog/src/tabs/tabbed-content.js

```diff
@@ -0,0 +1,89 @@
+import { TabBar, Tab } from '@dhis2-ui/tab'
+import PropTypes from 'prop-types'
+import React, { useState } from 'react'
+import { AccessAdd } from '../access-add/index.js'
+import { AccessList } from '../access-list/index.js'
+import { CascadeSharing } from '../cascade-sharing/index.js'
+import i18n from '../locales/index.js'
+
+export const TabbedContent = ({
+    type,
+    users,
+    groups,
+    publicAccess,
+    allowPublicAccess,
+    id,
+    onAdd,
+    onChange,
+    onRemove,
+}) => {
+    const [activeTabIndex, setActiveTabIndex] = useState(0)
+
+    if (type === 'visualization') {
```

Comment:
Could be a constant that could also be re-used in the prop types definition

Reply:
Fixed in 724e30abe69e1b27e62a7a31ac446ebcd3cf1a4d